cli/lib/kontena/cli/nodes/reset_command.rb

```diff
@@ -0,0 +1,31 @@
+module Kontena::Cli::Nodes
+  class ResetCommand < Kontena::Command
+    include Kontena::Cli::Common
+    include Kontena::Cli::GridOptions
+
+    requires_current_master
+    requires_current_master_token
+    requires_current_grid
+
+    parameter "NODE", "Node name"
+
+    option ["--token"], "TOKEN", "Use given node token instead of generating a random token"
+    option ["--clear-token"], :flag, "Clear token"
```

Comment:
This is intended as a rollback when unable to reconfigure a pre-existing node to use the node token after using `kontena node reset` to generate a node token.